kaitaistruct.py

```diff
@@ -30,22 +31,30 @@ def close(self):
 
     @classmethod
     def from_file(cls, filename):
-        f = open(filename, 'rb')
-        try:
+        with open(filename, 'rb') as f:
```

Comment:
You may also wanna look at #25

Reply:
I see that's a thing to consider. Thank you.